docker/bootstrap/Dockerfile.percona57

```diff
@@ -11,7 +11,7 @@ RUN apt-key adv --keyserver ha.pool.sks-keyservers.net \
     } | debconf-set-selections && \
     apt-get update && \
     apt-get install -y --no-install-recommends \
-        percona-server-server-5.7 libperconaserverclient18.1-dev && \
+        percona-server-server-5.7=5.7.14* libperconaserverclient18.1-dev && \
```

Comment:
Is there a reason why you want to pin it to a specific minor version?

Wouldn't it be better if we don't pin it by default? This would be less surprising: a rebuild of the image would always get you the latest version.

We regularly rebuild and test the images. Therefore, I'm not concerned about regressions between minor versions.